package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "lancer-longrim-data",
+  "version": "1.0.11",
+  "description": "Player-facing COMP/CON data from 'The Long Rim: a Lancer Setting', a LANCER setting for the LANCER TTRPG",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/OmnesPotens/long-rim-data.git"
+  },
+  "directories": {
+    "lib": "lib"
+  },
+  "author": "OmnesPotens",
+  "license": "GPL-3.0-or-later",
+  "bugs": {
+    "url": "https://github.com/OmnesPotens/long-rim-data/issues"
+  },
+  "homepage": "https://github.com/OmnesPotens/long-rim-data",
+  "scripts": {
+    "build-gpr": "node ./.github/gpr-publishing.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "lancer",
+    "nrfaw",
+    "no room for a wallflower",
```

Comment:
Heads up: These Wallflower keywords may be inappropriate for The Long Rim.

Reply:
Fixed